engines/sci/graphics/palette32.cpp

```diff
@@ -206,17 +209,25 @@ int16 GfxPalette32::matchColor(const byte r, const byte g, const byte b, const i
 	return bestIndex;
 }
 
-void GfxPalette32::updateForFrame() {
+bool GfxPalette32::updateForFrame() {
 	applyAll();
 	_versionUpdated = false;
 	// TODO: Implement remapping
```

Comment:
As far as I know, remapping was not reversed for SCI32 (though it is for SCI16) and, based on my glancing over the two, is not implemented exactly the same way as in SCI16. I could try to make the SCI16 implementation work with changes, though it sounded like @m-kiewitz was thinking keeping SCI32 graphics code separate was the best idea since the SCI16 code already has a lot of complexity.

Most notably, it seems that the SCI16 remap code only supports one remap at a time, but SCI32 lets developers set up to 9 separate remaps, and each separate remap has its own independent state about what percent, the range, the type, its own remap map, et cetera. It also operates during the draw routines which are totally different in SCI32, which would make the old remapping code difficult to just re-enable. So, I sorry, I know I am saying “no” a lot but I think the answer here is again “no”. :(